docs/src/components/home/GoldSponsors.tsx

```diff
@@ -63,6 +63,20 @@ const GOLDs = [
     description: 'Instagram followers, likes, views, and comments.',
     href: 'https://goread.io/?utm_source=MUI&utm_medium=referral&utm_content=homepage',
   },
+  {
+    src: '/static/sponsors/buzzoid-square.svg',
+    srcSet: '/static/sponsors/buzzoid-square.svg 3x',
+    name: 'Buzzoid',
+    description: 'Instant delivery Instagram followers.',
+    href: 'https://buzzoid.com/buy-instagram-followers/',
+  },
+  {
+    src: '/static/sponsors/twicsy-square.svg',
+    srcSet: '/static/sponsors/twicsy-square.svg 3x',
+    name: 'Twicsy',
+    description: 'Instant delivery Instagram followers.',
+    href: 'https://twicsy.com/buy-instagram-followers',
+  },
```

Comment:
I think this goes against the purpose: https://www.notion.so/mui-org/sales-Introduce-backlink-sponsor-plan-49e70924423e462aad099df16273c094?pvs=4#9f00df515854496697e45df38e913c73

```suggestion
```

or in a mini format.